### PR TITLE
docs: Technical Overview

### PR DESCRIPTION
**Pull request by changangus**
## Description

Documentation for:
- Tech Stack
- CI/CD

### Spec

See Story: [FSA21V2-153](https://sparkbox.atlassian.net/browse/FSA21V2-153)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
+- [Jest](https://jestjs.io/)
+- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/)
 
-Quickly get started using [Create Next App](https://github.com/vercel/next.js/tree/canary/packages/create-next-app#readme)!
+#### CI/CD:
 
-In your terminal, run the following command:
+- [Netlify](https://docs.netlify.com/?_gl=1%2a9cbevu%2a_gcl_aw%2aR0NMLjE2MzcxNzk2MTAuQ2p3S0NBaUE3ZEtNQmhCQ0Vpd0FPX2NyRkpkcTd0Uk1VQk9feHF4dHIwRENEZTZvSjdDRXo2dGNCUm5sVGo2cUE5NHJWVXluR05CdGt4b0N6ZzBRQXZEX0J3RQ..&_ga=2.212470036.1605752707.1637179610-1559964017.1632236220&_gac=1.85689835.1637179610.CjwKCAiA7dKMBhBCEiwAO_crFJdq7tRMUBO_xqxtr0DCDe6oJ7CEz6tcBRnlTj6qA94rVUynGNBtkxoCzg0QAvD_BwE) is used for automatic deployments and deployment previews. 
+- [Github Actions](https://docs.github.com/en/actions) allows us to make sure all tests and linters pass for each PR.
+  - [Sparkbox eslint config](https://github.com/sparkbox/eslint-config-sparkbox) is used to setup linting rules for javascript.
```
Comment (jonoliver) [inline README.md:16]: Minor grammatical nitpick: I believe `setup` is a noun, and the verb form is `set up`. Maybe just rephrase this to something like:
"is used to enforce JavaScript code styles"

**Review by jonoliver — CHANGES_REQUESTED** [review]
Hey Angus, just a small requested update on this one as well. Thanks!

File: README.md [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
+- [Next.js](https://nextjs.org/)
+- [Sass Modules](https://nextjs.org/docs/basic-features/built-in-css-support#sass-support)
+- [Jest](https://jestjs.io/)
+- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/)
 
-Quickly get started using [Create Next App](https://github.com/vercel/next.js/tree/canary/packages/create-next-app#readme)!
+#### CI/CD:
 
-In your terminal, run the following command:
+- [Netlify](https://docs.netlify.com/?_gl=1%2a9cbevu%2a_gcl_aw%2aR0NMLjE2MzcxNzk2MTAuQ2p3S0NBaUE3ZEtNQmhCQ0Vpd0FPX2NyRkpkcTd0Uk1VQk9feHF4dHIwRENEZTZvSjdDRXo2dGNCUm5sVGo2cUE5NHJWVXluR05CdGt4b0N6ZzBRQXZEX0J3RQ..&_ga=2.212470036.1605752707.1637179610-1559964017.1632236220&_gac=1.85689835.1637179610.CjwKCAiA7dKMBhBCEiwAO_crFJdq7tRMUBO_xqxtr0DCDe6oJ7CEz6tcBRnlTj6qA94rVUynGNBtkxoCzg0QAvD_BwE) is used for automatic deployments and deployment previews.
```
Comment (jonoliver) [inline README.md:14]: Let's link to the dashboard for this project for the Netlify link: https://app.netlify.com/sites/sb-apprentices/overview

Comment (jonoliver) [reply README.md:14]: Could we also update the links to use [footnote syntax](https://github.com/sparkbox/standard/tree/main/code-style/markdown#style) instead please?

**Review by jonoliver — CHANGES_REQUESTED** [review]
Nice, Angus! Just a bit of feedback on the links.

File: README.md (hunk 2 above)
Comment (changangus) [reply README.md]: Just updated the READMe to use footnote syntax and changed the link to the dashboard for the project!

**Review by jonoliver — APPROVED** [review]
